data-raw/process_data_sets_meredith.R

```diff
@@ -0,0 +1,143 @@
+library(tidyverse)
+library(stringr)
+library(lubridate)
+
+# ahca_polls ----------------------------------------------------------------------
+ahca_polls <- read_csv("data-raw/ahca-polls/ahca_polls.csv")
+# snake_case for variable names
+colnames(ahca_polls) <- colnames(ahca_polls) %>%
```

Comment:
Could you replace all `colnames()` code bits with pipes into the `clean_names()` function from the `janitor` package? i.e. make lines 6-8 here:
```
ahca_polls <- read_csv("data-raw/ahca-polls/ahca_polls.csv") %>% 
  clean_names()
```